handbook/pull-requests.md

```diff
@@ -0,0 +1,29 @@
+## Pull Requests
+
+Work on the data engineering team is most often delivered via a [pull request](communication.md#everything-starts-with-a-pull-request), or PR, allowing for discussions around a actionable proposed solution to a problem.
+
+### Submitting Pull Requests
+
+Unless [indicated otherwise](pull-requests.md#draft-pull-requests), when you submit a PR, you are acknowledging that the PR can be safely deployed to production at any time, by any individual with write access to the associated GitHub repository. 
+
+You may also add a `DO NOT MERGE` [label](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) to the PR in instances where you want to defer the deployment of the PR. An example of such an instance would be when a deliverable piece of work requires multiple interdependent PRs to be merged and deployed in a particular order.
+
+#### Linking PRs to JIRA
+
+In order for JIRA to detect that a ticket has a linked PR, you must do one of the following:
+
+* Give your development branch a name that begins with the JIRA ticket identifier (e.g. `DI-123_add-columns`).
```

Comment:
I think I'd prefer that we name (or rename) the dev branch by the ticket ID. It's much nicer to have a human-readable title of the PR, especially if it does not completely resolve the whole ticket.